CuratedContent/SelfPerceptions.md

```diff
@@ -0,0 +1,55 @@
+
+# A Look into Self-Perceptions about Software Engineering in the Scientic Community
+
+#### Contributed by [Rinku Gupta](https://github.com/rinkug "Rinku Gupta Github Profile")
+
+Researchers and scientists are now spending increasing amounts of
+time developing software. But how many of them are actually using
+software engineering practices that can help them write better software?
+
+Resource information | Details
+:--- | :--- 
+Paper title  | Self-Perceptions about Software Engineering: A Survey of Scientists and Engineers
+Authors | Jeffrey Carver, Dustin Heaton, Lorin Hochstein, and Roscoe Bartlett
+Publication | Year 2013, Computing in Science and Engineering - Vol 15, DOI: [10.1109/MCSE.2013.12](https://www.computer.org/csdl/magazine/cs/2013/01/mcs2013010007/13rRUx0xPqU)
+
+The journal article *Self-Perceptions about Software Engineering:
+A Survey of Scientists and Engineers*, is a survey paper published
+in 2013, that made observations that continue to be very relevant
+for the scientific software community even today. The article recognizes
+that scientists and engineers must often double as part-time software
+developers, with activities ranging from writing small-scale scripts
+for their research to highly complex code that will be used on top
+supercomputers of the world. But, while software development skills
+are becoming as integral to science and engineering as much as laboratory
+skills, the community hasn't developed a culture of training for
+the relevant software skills.
+
+The paper surveys close to 150 researchers, encompassing people
+from a rich educational background (with 82% respondents having a
+PhD, which is one of the highest globally recognized degrees for
+students) from the domain of mathematics and statistics, engineering,
+and computer science. Also relevant is the fact, is that the survey
+tries to analyze how respondents spent their time fractions - whether
```

Comment:
Change to "Also, the survey tries to analyze ..."?

Reply:
Changes done